storehouse/stock.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Start functions for storehouse
+ */
+
+let carPartsInStock = ['alternator', 'klocki hamulcowe'];
```

Comment:
Dobrze zacząłeś od utworzenia tablicy, ale nadałeś jej bardzo specyficzną nazwę.
To znaczy, że nasz program będzie dedykowany dla magaznu z częściami samocchodowymi, a co jeśli nasz klient zmieni branżę?
Nazwa powinna być bardzo mocno ogólna, abstrakcyjna, metaforyczna.

Ta tablica działa troche jak baza danych w obecnej chwili.

Kolejny błąd jest taki, że zainicjowałeś stan magazynowy z dwoma elementami na start.
To znaczy, że przy każdym świeżym uruchomieniu będziemy mieć dodatni stan magazynowy.

Oczywiście inicjacja stanów magazynowych powinna być stanem zerowym, czyli pustą tablicą.